src/Game.ts

```diff
@@ -14,6 +14,7 @@ class Game {
 
     afterMove(field: Field, move: string) {
         this.movePiece(field, move);
+        this.isCheck(this.turn);
```

Comment:
Nie musimy tego parametru przekazywać tutaj, z tego this.turn mozemy korzystać w funkcji, więc moze tego nie przekazujmy